<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcArtPhotoUpload</class>
 <widget class="QWidget" name="MusicLrcArtPhotoUpload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>上传歌手写真</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>460</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>上传歌手写真</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="artNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>62</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>歌手：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artSearchEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>62</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>305</y>
     <width>456</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>选择的图片尺寸过小，请重新选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="MusicLrcArtPhotoLabel" name="imageLabel" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>385</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>点击选择图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="introduceLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请上传尺寸1033*660以上的图片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>460</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>artNameLabel</zorder>
  <zorder>artSearchEdit</zorder>
  <zorder>stateLabel</zorder>
  <zorder>uploadButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>imageLabel</zorder>
  <zorder>selectButton</zorder>
  <zorder>introduceLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicLrcArtPhotoLabel</class>
   <extends>QWidget</extends>
   <header>musiclrcartphotolabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
